contracts/profiles/PreventsZWC.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+abstract contract PreventsZWC {
```

Comment:
Is there a way to make this upgradeable in case we want to change it later?
I guess we could just upgrade the parent contract, but curious if we can just update this abstract modifier in place.

Reply:
I looked into this. Since the contract is abstract, it cant be deployed directly, so I don't think it's possible to make it upgradeable. I think making the registrar upgrade is the only option (either moving this modifier directly into BNSRegistrar or changing this file, and then redeploying the Registrar)